osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -30,6 +30,8 @@ public class PlaySongSelect : SongSelect
         protected readonly BeatmapDetailArea BeatmapDetails;
         private bool removeAutoModOnResume;
 
+        private GameBeatmap beatmap;
```

Comment:
Are you sure we want yet another bindable here, rather than making `SongSelect`'s (or `OsuScreen`'s) one `protected`?